### PR TITLE
Add MuseScoreIE.

### PR DESCRIPTION
**Pull request by Ashish0804**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/ytdl-org/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/ytdl-org/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/yt-dlp/yt-dlp/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into yt-dlp each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [ ] Improvement
- [x] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

Closes https://github.com/yt-dlp/yt-dlp/issues/911

Ln 46-47 is such that ln 48 it will break if id is matched from user provided url instead of url from webpage, this is intended.
mp3_url is also extracted in such way that it should break if json is irregular.

### CODE REVIEW

File: yt_dlp/extractor/musescore.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+            'url': mp3_url,
+            'ext': 'mp3',
+            'vcodec': 'none',
+        }]
+
+        return {
+            'id': id,
+            'formats': formats,
+            'title': self._og_search_title(webpage),
+            'description': self._og_search_description(webpage),
```
Comment (nao20010128nao) [inline yt_dlp/extractor/musescore.py:60]: I don't think this is accurate extraction; it should give you `Download and print in PDF or MIDI free sheet music for Its A Small World Theme by Misc Soundtrack arranged by <uploader> for <instrument> (<formation>)` according to its HTML

Comment (Ashish0804) [reply yt_dlp/extractor/musescore.py:60]: if the uploader/site have set the wrong description then nothing we can do imo.
this seemed like the fastest way to get a description to me

Comment (Ashish0804) [reply yt_dlp/extractor/musescore.py:60]: i just checked on your url, and it gives description in that format with this code
So yeah this is a site issue

**Comment by Ashish0804** [comment]
Reasons to change .gitignore:
Official ytdl cookies guide says to use this [extn](https://chrome.google.com/webstore/detail/get-cookiestxt/bgaddhkoddajcdgocldbbfleckgcbcid/)  which saves in the filename format `sitename_cookies.txt`
While the file can be renamed to `cookies.txt`, for devs working with multiple sites at same time, it becomes annoying. All cookies can't be dumped in one file bcoz during development and debugging cookies are often needed to be redownloaded.
Therefore adding `*` should fix this.

**Pull request merged**